doozer/doozerlib/cli/scan_osh.py

```diff
@@ -158,6 +433,16 @@ async def run(self):
 
         self.trigger_scans(nvrs_for_scans)
 
+        if not self.runtime.group_config.scanning.jira_integration.enabled:
+            self.runtime.logger.info(f"Skipping OCPBUGS creation workflow since not enabled in group.yaml for "
```

Comment:
```suggestion
            self.runtime.logger.info(f"Skipping OCPBUGS creation workflow since not enabled in group.yml for "
```

Reply:
fixed in https://github.com/openshift-eng/art-tools/pull/141/commits/e4031289cbb0f229b3524b08cc04909ef204bc17

Reply:
I don't see it, did we miss it :)

Reply:
I ran pytest locally and it wiped out my doozer directory. Dont know what happened. It was just before I was about to push to GitHub. So had to make the changes from memory and I missed it. Sorry about that

Reply:
added https://github.com/openshift-eng/art-tools/pull/141/commits/6f3bc633c8c3bcb85742eb30ef2e4a68c0a5d861